app/graphql/mutations/stop_impersonating.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+module Mutations
+  class StopImpersonating < BaseMutation
+    field :success, Boolean, null: false
+
+    def authorized?
```

Comment:
I think this is probably overly strict: if `true_user != current_user`, then you should be able to stop impersonating.  Stopping impersonation always drops privileges, so it's always a conservative action to take...as long as we're careful that the only way `true_user` can become set is via `ImpersonateUser`, which in turn does the correct access checks.